app/components/edit_channel_info/index.js

```diff
@@ -53,40 +53,37 @@ export default class EditChannelInfo extends PureComponent {
         editing: false,
     };
 
+    constructor(props) {
+        super(props);
+
+        this.nameInput = React.createRef();
+        this.urlInput = React.createRef();
+        this.purposeInput = React.createRef();
+        this.headerInput = React.createRef();
+        this.lastText = React.createRef();
+        this.scroll = React.createRef();
+    }
+
     blur = () => {
-        if (this.nameInput) {
-            this.nameInput.blur();
+        if (this.nameInput?.current) {
```

Comment:
I thought `this.nameInput` could not be null and `?` may not be necessary, isn't it?

Reply:
Is just a fail safe and it does not hurt to have it and current can actually be null